### DISCUSSION TITLE
blog/20231213/

### DISCUSSION THREAD

**Discussion by giscus** [xaoxuu/blog-comments#37] [Announcements]
# blog/20231213/

废话不多说直接上链接，包含：iOS、iPadOS、macOS、tvOS、visionOS、watchOS 的组件库、设计模版、官方字体、SF 符号、苹果全系列产品模型图、商标等资源下载。

https://xaoxuu.com/blog/20231213/

**Comment by penndu** [comment]
评论测试~
